packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -73,48 +68,67 @@ function ShoppingPane(props) {
   const speak = useLiveRegion('assertive');
   const [loaded, setLoaded] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
```

Comment:
@swissspidy There is now a lot of state here. Wonder if we should refactor this later to be a little cleaner.

Reply:
Doesn't seem like too much right now. Maybe something for the future when we add more stuff here—can keep an eye on it.